### PR TITLE
Feature/ Implement Select component

### PR DESCRIPTION
**Pull request by JennyDoJD**
Result (UPDATED):
image

Ticket: [Implement Select common component](https://trello.com/c/6FDEnfpa/32-implement-select-common-component)

### CODE REVIEW

File: practice/react-practice-one/src/components/Select/Select.jsx [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+/* Import components */
+import SelectIcon from '../Icons/SelectIcon';
+
+const Select = ({ options, onChange }) => {
+  return (
+    <div className="select-overlay ">
+      <SelectIcon />
+      <select className="sort-container" onChange={onChange}>
+        <option value="default">Sort by</option>
```
Comment (ghost) [inline practice/react-practice-one/src/components/Select/Select.jsx]: How can we use this component in other cases?

Comment (JennyDoJD) [reply practice/react-practice-one/src/components/Select/Select.jsx]: I fixed it

File: practice/react-practice-one/src/components/Select/Select.jsx [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+      {icon && icon}
+      <select className="sort-container" onChange={onChange}>
+        <option value="default">{text}</option>
+        {options.map((group) => (
+          <optgroup key={group.label} label={group.label}>
+            {group.items.map((item) => (
+              <option
+                key={item.key}
+                value={item.value}
+                data-sort-by={item.sortBy}
```
Comment (ghost) [inline practice/react-practice-one/src/components/Select/Select.jsx]: why do we need this property?

Comment (JennyDoJD) [reply practice/react-practice-one/src/components/Select/Select.jsx]: key={item.key}: used to optimize the rendering process.
value={item.value}: used to determine the value of the option when the user selects.
data-sort-by={item.sortBy}: used to handle the sorting logic.

Comment (ghost) [reply practice/react-practice-one/src/components/Select/Select.jsx]: @JennyDoJD `data-sort-by` we don't need this, you should mange it by using state

Comment (JennyDoJD) [reply practice/react-practice-one/src/components/Select/Select.jsx]: Got it. I am fixing it

**Review by ghost — APPROVED** [review]

**Pull request merged**